internal_data/regexes.R

```diff
@@ -3,6 +3,7 @@
 # > This should contain only (ASCII) letters, numbers and dot,
 # > have at least two characters and start with a letter and not end in a dot.
 CANONICAL_PACKAGE_NAME_RE = '[a-zA-Z]{1}[a-zA-Z0-9\\.]*[a-zA-Z0-9]{1,}'
+CANONICAL_PACKAGE_NAME_RE_STANDALONE = '^[a-zA-Z]{1}[a-zA-Z0-9\\.]*[a-zA-Z0-9]{1,}$'
```

Comment:
Can the definition of `CANONICAL_PACKAGE_NAME_RE_STANDALONE` instead be defined using `CANONICAL_PACKAGE_NAME_RE` (i.e. like `paste0('^', CANONICAL_PACKAGE_NAME_RE, '$')` or the `sprintf` equivalent)?

This will make fixing bugs in the regexes simpler if it turns out `CANONICAL_PACKAGE_NAME_RE` is buggy (or if CRAN updates this never updated policy)

Reply:
Addressed in 37ee7e8582947b29bf122ed29315ebd870f7d090